next-sitemap.config.js

```diff
@@ -28,29 +28,37 @@ module.exports = {
 		'sitemap.xml',
 		'/server-sitemap.xml',
 		'/boundary/docs*',
+		'/boundary/tutorials*',
```

Comment:
This excludes list will grow until as we roll out new content types for server side sitemap creation. After all content types are added, most, if not all of the product page paths will be removed.